Makefile

```
@@ -94,6 +95,16 @@ build-docker:
 		.
 	@echo "Docker image built: odg-core:$(ODG_CORE_LIBS_VERSION)"
 
+# Run PostgreSQL database instance
+run-db:
+	@echo "Starting PostgreSQL database instance..."
+	@echo "Database port: $(DB_PORT)"
+	@docker run -dit --name postgres \
+        -e "POSTGRES_USER=$(DB_USER)" \
+        -e "POSTGRES_PASSWORD=$(DB_PASSWORD)" \
+		-p "$(DB_PORT):5432" \
+		postgres:16
```

checkmake (0.3.2)

[warning] Target "postgres" should be declared PHONY.

(phonydeclared)